hugr-core/src/hugr/serialize/test.rs

```diff
@@ -79,7 +79,7 @@ macro_rules! include_schema {
             static ref $name: NamedSchema =
                 NamedSchema::new("$name", {
                     let schema_val: serde_json::Value = serde_json::from_str(include_str!(
-                        concat!("../../../../specification/schema/", $path, "_v2.json")
+                        concat!("../../../../specification/schema/", $path, "_v4.json")
```

Comment:
`v3` is not even merged (#1352) and we are pushing for `v4` already? :P

Reply:
yes this is assuming 1352 goes in first

Reply:
can we just include this in the v3 change?

Reply:
have reverted schema changes pending #1352

Reply:
👍 
If this goes in the same release we should bundle do a single version bump.